module/metrics.go

```diff
@@ -400,7 +400,17 @@ type ExecutionMetrics interface {
 	ExecutionBlockDataUploadFinished(dur time.Duration)
 }
 
+type BackendScriptsMetrics interface {
+	// Record the round trip time while executing a script
+	ExecuteScriptRTT(dur time.Duration, size int)
```

Comment:
Maybe use `ScriptExecuted` to align with the naming convention used by other metrics